ZTNA APIs [Public Preview] (#21578)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/networkaccess-conditionalaccesssettings-get.md

---
title: "Get conditionalAccessSettings"

> description: "Read the  conditional access signaling for source IP restoration and Connectivity through NaaS"

Reviewer: I recommend avoid NaaS because we don't use the term anywhere to refer to Global Secure Access nor have we agreed to use it as the TOC name for this set of API docs.
```suggestion
description: "Read the details of conditional access signaling for source IP restoration."
```

author: Moti-ba
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: apiPageType
---

# Get conditionalAccessSettings
Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Read the  conditional access signaling for source IP restoration and Connectivity through NaaS

Reviewer: Match with the metadata description.

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|NetworkAccess.Read.All|
|Delegated (personal Microsoft account)| Not supported|
|Application|NetworkAccess.Read.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /networkAccess/settings/conditionalAccess
```

## Optional query parameters

> This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).

Reviewer: Document supported query parameters. Can be $select only (if supported). If not, then "This method does not support any OData..."
